### PR TITLE
Accessibility Day Preparations

### PR DESCRIPTION
**Pull request by matt-paul**
### Description

Add greyscale option to the red badger website to mark accessibility day.  This is a duplication of the code used on the blog for the same purpose.

### CODE REVIEW

File: site/index.ejs [hunk 1]
```diff
@@ -36,6 +36,74 @@
         }
     </script>
     <title><% if (typeof title !== 'undefined') {%><%= title %><% } %></title>
+
+    <script>
+    var colourOnButtonText = 'Make my day accessible';
```
Comment (dpiatek) [inline site/index.ejs]: Looks like the indentation is a bit off here

Comment (matt-paul) [reply site/index.ejs]: thanks, updated!

File: site/index.ejs [hunk 2]
```diff
@@ -36,6 +36,74 @@
         }
     </script>
     <title><% if (typeof title !== 'undefined') {%><%= title %><% } %></title>
+
+    <script>
+      var colourOnButtonText = 'Make my day accessible';
```
Comment (dpiatek) [inline site/index.ejs]: One more thing @matt-paul - could we add a comment here explaining what this is and how it's supposed to work with the other (button) PR and that there is a similar script in blog? Just conscious of people maybe doing this next year and it not necessarily being clear on what is needs to be done.

Comment (matt-paul) [reply site/index.ejs]: 👍

**Pull request merged**